### PR TITLE
Add initial rabbitmq docs

### PR DESCRIPTION
**Pull request by tianon**
https://github.com/docker-library/official-images/pull/381

### CODE REVIEW

**Comment by Moghedrin** [comment]
LGTM

File: rabbitmq/content.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+`/var/lib/rabbitmq` a volume by default.
+
+### Management Plugin
+
+There is a second set of tags provided with the [management
+plugin](https://www.rabbitmq.com/management.html) installed and enabled by
+default, which is available on the standard management port of 15672, with the
+default username and password of `guest` / `guest`:
+
+    docker run -d -e RABBITMQ_NODENAME=my-rabbit --name some-rabbit rabbitmq:3-management
```
Comment (yosifkit) [inline rabbitmq/content.md]: Might want the same sort of info about localhost and container-ip to access it like [rails](https://github.com/docker-library/docs/blame/master/rails/content.md#L31-37)

Comment (tianon) [inline rabbitmq/content.md]: Good point; updated!

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**